test/end2end_test.go

```diff
@@ -888,6 +888,31 @@ type lazyConn struct {
 	beLazy int32
 }
 
+// possible conn closed errors.
+const possibleConnResetMsg = "connection reset by peer"
+const possibleEOFMsg = "error reading from server: EOF"
+
+// isConnClosedErr checks the error msg for possible conn closed messages. There
+// is a raceyness in the timing of when TCP packets are sent from client to
+// server, and when we tell the server to stop, so we need to check for both of
+// these possible error messages:
+//  1. If the call to ss.S.Stop() causes the server's sockets to close while
+//     there's still in-fight data from the client on the TCP connection, then
+//     the kernel can send an RST back to the client (also see
+//     https://stackoverflow.com/questions/33053507/econnreset-in-send-linux-c).
+//     Note that while this condition is expected to be rare due to the
+//     rpcStartedOnServer synchronization, in theory it should be possible,
```

Comment:
Mentioning `rpcStartedOnServer` in this comment might no longer be appropriate because it is used only `TestDetailedConnectionCloseErrorPropagatesToRpcError`. But `isConnClosedErr` is also used from other places now. So, this comment might need to updated a little to apply for all usages of this function.

Reply:
i missed that. Let me update that